integration_tests/mockApis/responses/expectedArrivals.ts

```diff
@@ -71,7 +71,7 @@ export default [
     fromLocationType: 'COURT',
   },
   {
-    id: '12345',
+    id: '00000-11111',
```

Comment:
Just to make a clearer distinction between move id and prisoner numbers used in the tests